ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+    post:
+    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-deprovision
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      CREDENTIALS_MODE: Passthrough
+      OO_PACKAGE: gcp-filestore-csi-driver-operator
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-cluster-csi-drivers
+      OO_TARGET_NAMESPACES: '!all'
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.14
```

Comment:
Hard code 4.14 here? is it expected?

Reply:
I think we should move this setting to job config level just like what `gcp-ipi-compact-filestore-csi` did.